DuckDuckGo/NavigationBar/View/NavigationBarViewController.swift

```diff
@@ -191,6 +192,12 @@ final class NavigationBarViewController: NSViewController {
 #endif
 
         subscribeToAIChatOnboarding()
+
+        if #available(macOS 14.4, *) {
+            WebExtensionManager.shared.toolbarButtons().enumerated().forEach { (index, button) in
+                menuButtons.insertArrangedSubview(button, at: index)
```

Comment:
The button insertion affects minimum window width. This can potentially lead to the window not fitting onto a screen breaking core user experiences, so I‘d say this should be fixed.
I think the address bar center anchor priority should be lowered so it‘s moved left when buttons don‘t fit. 
Ideally there should be an overflow menu, but this is probably out of scope.

Reply:
Good idea 👍

Reply:
Another note related to the button: extensions aren‘t loaded in a fire window, but the button is still added as if the extension is running, I think we shouldn‘t add the button to Fire windows

Reply:
And another one: when using Cookie BG Picker, its popover is not hidden when switching to another window. And when it‘s open in the background window you can‘t open another one in another window.